projects/models.py

```diff
@@ -101,14 +105,30 @@ def __str__(self):
         return self.title
 
 
+def add_project_controls(instance: Project):
+    try:
+        controls = apps.get_model("catalogs", "Controls")
+        catalog_controls = controls.objects.filter(catalog_id=instance.catalog)
+        print(catalog_controls)
+        if catalog_controls:
```

Comment:
We should consider using [exists](https://docs.djangoproject.com/en/4.1/ref/models/querysets/#exists) here:
```suggestion
        if catalog_controls.exists():
```